### ISSUE TITLE
Attention-Aware Generalized Mean Pooling for Image Retrieval

### ISSUE THREAD

**Issue by chullhwan-song** [chullhwan-song/Reading-Paper#157]
https://arxiv.org/abs/1811.00202

**Comment by chullhwan-song** [comment]
## PROPOSED METHOD
###  Network and Pooling
* [GeM](https://github.com/chullhwan-song/Reading-Paper/issues/153)
###  Attention-Aware GeM
* attention-aware GeM (**AGeM**) descriptor
* ResNet-101의 residual block & attention unit의 조합
![image](https://user-images.githubusercontent.com/40360823/66371354-ddd0be80-e9dd-11e9-9404-a0414303b65f.png)
    * 3개의 attention unit : Att1, Att2_1, Att2_2
       * Att1 : 4개의 conv layer
          * 3 × 3, 3 × 3, 1 × 1, and 1 × 1
            * 첫번째만 stride=2, 나머지 1
          * output dim은 1024, 512, 512, 2048
          * 마지막 layer를 제외한, BN  & ReLU activation 적용
          * 마지막 layer는 sigmoid function
       * Att2_1 & Att2_2 
          * 간단히, kernel size 1 × 1, stride 1 이루어진 하나의 conv layer로만 구성하고 input과 output의 dim이 같다. 그리고 sigmoid 적용
    * attention unit과 conv feature map과의 결합 
         * ⊗ : Hadamard product == element wise product
         * Fig.1 참조
    * final output
         *   element wise product 결합이 아닌, 마직막 layer에서는 attention unit과 conv feature map의 **+** 결합 
    * 이후, GeM > l2 normalization 적용 > 2048 dimension 크기의 최종 descriptor
###  Loss Function and Whitening
* imagenet pre-trained model > finetuning: labelled landmark images를 이용한 분류학습
* 이후, triplet loss, contrastive loss
* PCA

## 실험
![image](https://user-images.githubusercontent.com/40360823/66373803-736f4c80-e9e4-11e9-8850-31f9662efc62.png)
   * 수치가 이해되지 않는다. [DIR](https://github.com/chullhwan-song/Reading-Paper/issues/17)에서의 수치와 일치되지 않는다.(ctrl+f 로 찾을수 없다.ㅠ)
